paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsViewModel.kt

```diff
@@ -55,6 +56,7 @@ internal class PaymentOptionsViewModel @Inject constructor(
     lpmRepository: LpmRepository,
     savedStateHandle: SavedStateHandle,
     linkHandler: LinkHandler,
+    linkInteractor: LinkInteractor,
```

Comment:
This is confusing, there is a handler and interactor, what does what? It is pretty hard to tell here.